Cargo.toml

```diff
@@ -65,8 +65,13 @@ features = [
   "Win32_System_WindowsProgramming",  # General future used for various types/funcs
 ]
 
-[target.'cfg(target_os = "wasi")'.dependencies]
+[target.'cfg(all(target_os = "wasi", not(target_env = "p2")))'.dependencies]
 wasi = "0.11.0"
+
+[target.'cfg(all(target_os = "wasi", target_env = "p2"))'.dependencies]
+wasi = "0.13.3"
```

Comment:
Is this really the correct way to go about this? Only 0.11 supports p1?

Reply:
Looking at https://docs.rs/wasi/latest/wasi/
I would say yes, the last mention I see of `p1` is on `0.11`